central/notifiers/microsoftsentinel/sentinel.go

```diff
@@ -132,16 +185,11 @@ func (s sentinel) AlertNotify(ctx context.Context, alert *storage.Alert) error {
 			time.Sleep(wait * time.Millisecond)
 		}),
 	)
-
-	if err != nil {
-		return errors.Wrap(err, "failed to upload logs to azure")
-	}
-	return nil
 }
 
-func (s sentinel) prepareLogsToSend(alert *storage.Alert) ([]byte, error) {
+func (s sentinel) prepareLogsToSend(msg protocompat.Message) ([]byte, error) {
```

Comment:
nit: Since you switched the type and it is thus handling all data streams, I think a comment may be useful here.